_includes/faq.html

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <div class="faq">
+    <a name="faq"></a>
+    <h2>{{ site.data.translations.faq[page.lang] }}</h2>
+    {% assign sets = site.questions | where:"lang", page.lang | sort: "index" | group_by: "set" | sort: "name" %}
+    {% for set in sets %}
+    {% assign keyname = set.name | slice: 2, 999 | prepend: "faq-" %}
+    <input type="radio" name="faq" id="radio-{{ keyname }}"{% if keyname == "faq-top5" %} checked{% endif %}>
+    <label for="radio-{{ keyname }}"><h3>{{ site.data.translations[keyname][page.lang] }}</h3></label>
+    <div class="tab tab-{{ keyname }}">
+      {% for question in set.items %}
+      <details>
+        <summary><h4>{{ question.title }}</h4></summary>
```

Comment:
>  Because the <summary> element has a default role of button (which strips all roles from child elements), this example will not work for users of assistive technologies such as screen readers. The <h4> will have its role removed and thus will not be treated as a heading for these users.